From 9d9bbdf452ed1f7d8b6ecb35318f5ef48b9f7b1b Mon Sep 17 00:00:00 2001
From: Victor Fuentes <victor@vlinkz.dev>
Date: Wed, 29 Jul 2026 00:24:52 -0700
Subject: [PATCH 061/165] clk: qcom: dispcc-kaanapali: Don't park the MDP RCG
 at registration

Parking it breaks the bootloader's running display, so use
clk_rcg2_shared_no_init_park_ops like x1e80100 and sc8280xp.

Signed-off-by: Victor Fuentes <victor@vlinkz.dev>
---
 drivers/clk/qcom/dispcc-kaanapali.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/qcom/dispcc-kaanapali.c b/drivers/clk/qcom/dispcc-kaanapali.c
index f8832482bd7a..d4301edde7d3 100644
--- a/drivers/clk/qcom/dispcc-kaanapali.c
+++ b/drivers/clk/qcom/dispcc-kaanapali.c
@@ -688,7 +688,7 @@ static struct clk_rcg2 disp_cc_mdss_mdp_clk_src = {
 		.parent_data = disp_cc_parent_data_9,
 		.num_parents = ARRAY_SIZE(disp_cc_parent_data_9),
 		.flags = CLK_SET_RATE_PARENT,
-		.ops = &clk_rcg2_shared_ops,
+		.ops = &clk_rcg2_shared_no_init_park_ops,
 	},
 };
 
-- 
2.56.0

